crypto/dsa/dsa_ameth.c

```diff
@@ -361,6 +365,10 @@ static int do_dsa_print(BIO *bp, const DSA *x, int off, int ptype)
             <= 0)
             goto err;
     }
+    else{
```

Comment:
Nit. Merge the closing brace and newly provided one together and add the missing space between else and the opening brace.

Reply:
Hi @beldmit ,
Please review the code with above mentioned changes.

Reply:
LGTM